protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -494,6 +494,13 @@ service TradingDataService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Liquidity providers"};
   }
 
+  // List paid liquidity fees data
+  //
+  // List information about paid liquidity fees for a given market, or asset id.
+  rpc ListPaidLiquidityFees(ListPaidLiquidityFeesRequest) returns (ListPaidLiquidityFeesResponse) {
```

Comment:
Is there a reason this is a new API? The ticket from frontend mentions adding this information to the existing rewards API:
"""
    /api/v2/rewards
    /api/v2/rewards/epoch/summaries
    /api/v2/rewards/summaries (Less important ... I have no idea if anyone uses this)
"""

Reply:
yea the ticket was wrong around that, what we want here really is the equivalent of the feeStats endpoint for the lpFees